scrapers_next/az/people.py

```diff
@@ -0,0 +1,76 @@
+import lxml.html
+from spatula import HtmlListPage, CSS, SelectorError
+from ..common.people import ScrapePerson
+
+
+class BrokenHtmlListPage(HtmlListPage):
+    def postprocess_response(self) -> None:
+        # this page has a bad comment that makes the entire page parse incorrectly
+        fixed_content = self.response.content.replace(b"--!>", b"-->")
+        self.root = lxml.html.fromstring(fixed_content)
+        if hasattr(self.source, "url"):
+            self.root.make_links_absolute(self.source.url)  # type: ignore
+
+
+class LegList(BrokenHtmlListPage):
+    def process_item(self, item):
+        try:
+            name = CSS("a").match(item)[0].text_content()
+            if "--" in name:
+                name = name.split("--")[0].strip()
+
+            district = CSS("a").match(item)[1].text_content()
+
+            party = CSS("td").match(item)[2].text_content()
+            if party == "R":
+                party = "Republican"
+            elif party == "D":
+                party = "Democratic"
+
+            email = CSS("td").match(item)[3].text_content()
+            if email.startswith("Email: "):
+                email = email.replace("Email: ", "").lower() + "@azleg.gov"
+            else:
+                email = ""
+
+            room = CSS("td").match(item)[4].text_content()
```

Comment:
typically when we have a bunch of similar matches we do it just once and unpack the 6(?) tds, e.g.:
```
_, _, party, email, room, capitol_phone = CSS("td")
```

(not sure if [0] and [1] are used, but name them _ if not)

and then for each td you can process as you otherwise do here
```
party = party.text_content()
if party == "R": 
 ...
```